app/_hub/kong-inc/ai-proxy/overview/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+nav_title: Overview
+---
+
+The AI Proxy plugin lets you transform and proxy requests to a number of AI providers and models.
+
+The plugin accepts requests in one of a few defined and standardised formats, translates them to the configured target format, and then transforms the response back into a standard format.
+
+The AI Proxy plugin supports `v1/chat` and `v1/completion` style requests for all of the following providers:
```

Comment:
These route types should match the configurations and documentation elsewhere.   `llm/v1/chat` and `llm/v1/completion`